spec/mailers/notification_mailer_spec.rb

```diff
@@ -26,6 +26,29 @@
         expect(part.body.to_s).to include "guides/#{guide.id}/editions"
       end
     end
+
+    it "presents multi line comments correctly" do
+      comment_text = %{This guide sure could use more cow bell.
```

Comment:
There isn't a nice way to do this really. Some people roll their own thing that strips leading whitespace.. but.. seems a bit OTT for the benefit to me.

Ruby 2.3 has a thing but we're on 2.2. https://infinum.co/the-capsized-eight/articles/multiline-strings-ruby-2-3-0-the-squiggly-heredoc

Maybe we can upgrade? :shrug: